types/nodemailer-express-handlebars/nodemailer-express-handlebars-tests.ts

```diff
@@ -0,0 +1,29 @@
+import * as expressHandlebars from "express-handlebars";
+import { createTransport } from "nodemailer";
+import * as hbs from "nodemailer-express-handlebars";
```

Comment:
Looks like this should be `import hbs = require("nodemailer-express-handlebars");` to make `hbs` callable?